Casks/pokefinder.rb

```diff
@@ -0,0 +1,11 @@
+cask "pokefinder" do
+  version "4.0.0"
+  sha256 :no_check
+
+  url "https://github.com/Admiral-Fish/PokeFinder/releases/latest/download/PokeFinder-macOS.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/Admiral-Fish/PokeFinder/releases/download/v#{version}/PokeFinder-macOS.tar.gz"
```
Please use the versioned `url` and set the `sha256` above.